rfc8446.xml

```diff
@@ -4574,18 +4584,17 @@ The registry has been initially populated with the values described in
 rsa_pss_rsae_sha256, rsa_pss_rsae_sha384, rsa_pss_rsae_sha512,
 rsa_pss_pss_sha256, rsa_pss_pss_sha384, rsa_pss_pss_sha512, and ed25519.
 The "Recommended" column is assigned a value of "N" unless explicitly
-requested, and only IETF Standards Track RFCs can request a value of "Y".
-</t>
+requested, and adding a value with a "Recommended" value of "Y" requires Standards Action <xref target="RFC8126"/>.</t>
```

Comment:
Attempting to unify with the language I think we're ending up with for RFC 8447.